### PR TITLE
[ROX-12197] Remove alert and process tags

### PR DESCRIPTION
**Pull request by md2119**
## Description

During my usual end-of-sprint CHANGELOG checks, I found that the removal of support for violation tags and process tags in `3.72` was announced back in `3.69`. This PR removes both frontend and backend code.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [X] Evaluated and added CHANGELOG entry if required
- ~Determined and documented upgrade steps~
- ~Documented user facing changes~

If any of these don't apply, please comment below.

## Testing Performed
Before:
Violations page
image

Risk page
image

After:

Violations page
image

Risk page
image

### CODE REVIEW

**Review by vjwilson — APPROVED** [review]
Deleted code is the best code.

File: CHANGELOG.md [hunk 1]
```diff
@@ -60,6 +61,7 @@ authentication is still required for an email notifier, but the user can now cho
   - This means the configuration for the init container has been updated to include "POSTGRES_PASSWORD_FILE" and some volume mounts which are now required.
 - Debian 9 has reached EOL, so Scanner now marks Debian 9 images as stale.
   - The Debian Security Tracker has also stopped tracking Debian 9 vulnerabilities, so there will be no more new Debian 9 vulnerabilities.
+- Support for violation comments and process comments has been removed.
```
Comment (vikin91) [inline CHANGELOG.md:64]: I guess this is a duplicate. The feature was still there in 3.71, right?

Comment (md2119) [reply CHANGELOG.md:64]: Tags and comments are two different features. Comments features was removed in 3.71.

**Review by connorgorman — APPROVED** [review]

**Pull request merged**